README.rst

```diff
@@ -112,12 +112,62 @@ You can use the library to send and receive requests and responses like so,
 
         return blog.dumps(response)
 
+Note that this example sends and receives just the request/response payload. It
+does not wrap the payload in a message envelope as expected by Apache Thrift.
+If you want to send or receive standard Apache Thrift requests to talk to other
+Apache Thrift services, you have to use the ``loads.message`` and
+``dumps.message`` APIs. For example,
+
+.. code-block:: python
+
+    # client.py
+
+    def new_post():
+        post = blog.PostDetails(...)
+        request = BlogService.newPost.request(post)
+        payload = blog.dumps.message(request)
+        # ^ Instead of using blog.dumps, we use blog.dumps.message to indicate
+        # that we want the request wrapped in a message envelope.
+
+
+        response_payload = send_to_server(payload)
+
+        # Similarly, instead of using blog.loads, we use blog.loads.message to
+        # indicate that we want to parse a response stored inside a message.
+        response_message = blog.loads.message(BlogService, response_payload)
+        response = response_message.body
+
+        if response.unauthorized is not None:
+            raise response.unauthorized
+        else:
+            return response.success
+
+
+.. code-block:: python
+
+    # server.py
+
+    def handle_request(request_payload):
+        message = blog.loads.message(BlogService, request_payload)
+        if message.name == 'newPost':
+            request = message.body
+            # ...
+            response = BlogService.newPost.response(success=post_uuid)
+            return blog.dumps.message(response, seqid=message.seqid)
```

Comment:
What's the impact of omitting `seqid` when we have thriftrw on the other end? What about Apache?